Components/Maps/Maps.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, View, Text, Dimensions } from 'react-native';
+import MapView, { PROVIDER_GOOGLE } from 'react-native-maps';
+import { Agenda } from 'react-native-calendars'
+import { Marker } from 'react-native-maps';
+
+
+export const Maps = ({ route }) => {
+  const { destination, startDate, endDate, name } = route.params
+  console.log('lat/long: ', destination);
+  console.log('dates: ', startDate, endDate);
+
+  const [ marker, setMarker ] = useState({});
+
+  return (
+    <View style={styles.container}>
+      <MapView style={styles.mapStyle}
+        provider={PROVIDER_GOOGLE}
+        region={{
+          latitude: parseFloat(destination.lat),
+          longitude: parseFloat(destination.long),
+          latitudeDelta: 0.5,
+          longitudeDelta: 0.55
+        }}
+        showsUserLocation
+      >
+      <Marker 
+        coordinate={{
+          latitude: parseFloat(destination.lat),
+          longitude: parseFloat(destination.long),
```

Comment:
This parseFloat makes a big difference! In the future, it might be best to use this lat/long just to center the map and only display markers for the Yelp activities. But good work, it looks great!

Reply:
I like that that! I'm still looking into how to set up deltas dynamically for when we have multiple markers but a good fail safe is to just center on location (with no marker) and render anything that is within a certain radius.  Thanks John!